mscore/preferences.cpp

```diff
@@ -73,8 +73,10 @@ void Preferences::init(bool storeInMemoryOnly)
       {
             {PREF_APP_AUTOSAVE_AUTOSAVETIME,                       new IntPreference(2 /* minutes */, false)},
             {PREF_APP_AUTOSAVE_USEAUTOSAVE,                        new BoolPreference(true, false)},
-            {PREF_APP_KEYBOARDLAYOUT,                              new StringPreference("US - International")},
-            {PREF_APP_PATHS_INSTRUMENTLIST1,                       new StringPreference(":/data/instruments.xml", false)},
+	      {PREF_APP_KEYBOARDLAYOUT,                              new StringPreference("US - International")},
+	      {PREF_APP_PAGE_UNITS_GLOBAL,                           new BoolPreference(true, false)},
+	      {PREF_APP_PAGE_UNITS_VALUE,                            new IntPreference(0, false)},
+	      {PREF_APP_PATHS_INSTRUMENTLIST1,                       new StringPreference(":/data/instruments.xml", false)},
```

Comment:
These four lines have two extra spaces in front.

Reply:
Maybe not. It appeared that way in the diff.

Reply:
Ah, some of the spaces are actually tabs. You should correct that.

Reply:
That's odd.  They don't appear that way in VS2017.  I'll try opening them in another editor.  They do show up as indented in Notepad.  It's because there's tab character in there at the start of the line.  Fixed.